procedures: Add PostgreSQL migration guides (#2163)
codenvy/che-docs: modules/administration-guide/partials/proc_reinstalling-che.adoc

[id="reinstalling-{prod-id-short}_{context}"]
= Reinstalling {prod-short}

.Procedure

> . Remove current {prod} instance by following the instructions xref:installation-guide:uninstalling-che.adoc[].

Reviewer: ```suggestion
. Remove the current {prod-short} instance as explained in xref:installation-guide:uninstalling-che.adoc[].
```

> . Deploy a new {prod} instance by following the instructions xref:installation-guide:installing-che.adoc[].

Reviewer: ```suggestion
. Deploy a new {prod-short} instance as explained in xref:installation-guide:installing-che.adoc[].
```
